File: android/app/src/main/java/com/nvp/node/ui/theme/Theme.kt
package com.nvp.node.ui.theme

import androidx.compose.foundation.isSystemInDarkTheme
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Typography
import androidx.compose.material3.darkColorScheme
import androidx.compose.material3.lightColorScheme
import androidx.compose.runtime.Composable
import androidx.compose.ui.graphics.Color

// Fintech identity shared with the iOS app: deep teal surfaces + lime accent.
object Nvp {
    val accent = Color(0xFFC9F24E)      // lime — primary action
    val onAccent = Color(0xFF0E211B)
    val tealLight = Color(0xFF143A32)   // feature card (balance hero)
    val tealDark = Color(0xFF0B2A22)
    val onTeal = Color(0xFFE8F3EE)

    val bgLight = Color(0xFFF4F7F2); val bgDark = Color(0xFF0E1F1A)
    val cardLight = Color(0xFFFFFFFF); val cardDark = Color(0xFF14342B)
    val card2Light = Color(0xFFEDF2EC); val card2Dark = Color(0xFF1C4A3B)
    val textLight = Color(0xFF10231C); val textDark = Color(0xFFEAF3EE)
    val mutedLight = Color(0xFF5C7268); val mutedDark = Color(0xFF9DB8AC)
    val green = Color(0xFF12A05E); val red = Color(0xFFD64545)
}

// Convenience accessors that follow the active scheme.
val androidx.compose.material3.ColorScheme.muted: Color get() = onSurfaceVariant
val androidx.compose.material3.ColorScheme.card2: Color get() = surfaceVariant

private val LightScheme = lightColorScheme(
    primary = Nvp.accent, onPrimary = Nvp.onAccent,
    background = Nvp.bgLight, onBackground = Nvp.textLight,
    surface = Nvp.cardLight, onSurface = Nvp.textLight,
    surfaceVariant = Nvp.card2Light, onSurfaceVariant = Nvp.mutedLight,
    secondary = Nvp.tealLight, onSecondary = Nvp.onTeal,
    error = Nvp.red,
)
private val DarkScheme = darkColorScheme(
    primary = Nvp.accent, onPrimary = Nvp.onAccent,
    background = Nvp.bgDark, onBackground = Nvp.textDark,
    surface = Nvp.cardDark, onSurface = Nvp.textDark,
    surfaceVariant = Nvp.card2Dark, onSurfaceVariant = Nvp.mutedDark,
    secondary = Nvp.tealDark, onSecondary = Nvp.onTeal,
    error = Nvp.red,
)

@Composable
fun NvpTheme(dark: Boolean = isSystemInDarkTheme(), content: @Composable () -> Unit) {
    MaterialTheme(
        colorScheme = if (dark) DarkScheme else LightScheme,
        typography = Typography(),
        content = content,
    )
}

File: macos/NVPMac/Theme.swift
import SwiftUI

/// Shared NVP fintech identity (teal + lime), matching the iOS / Android apps.
enum Theme {
    static let bg = Color(red: 0.055, green: 0.122, blue: 0.102)     // #0E1F1A
    static let card = Color(red: 0.078, green: 0.204, blue: 0.169)   // #14342B
    static let card2 = Color(red: 0.110, green: 0.290, blue: 0.231)  // #1C4A3B
    static let teal = Color(red: 0.078, green: 0.227, blue: 0.196)   // #143A32
    static let accent = Color(red: 0.788, green: 0.949, blue: 0.306) // #C9F24E lime
    static let onAccent = Color(red: 0.055, green: 0.130, blue: 0.106)
    static let text = Color(red: 0.918, green: 0.953, blue: 0.933)   // #EAF3EE
    static let muted = Color(red: 0.616, green: 0.722, blue: 0.675)  // #9DB8AC
    static let green = Color(red: 0.310, green: 0.878, blue: 0.627)
    static let red = Color(red: 1.0, green: 0.42, blue: 0.42)
}

extension View {
    func nvpCard() -> some View {
        self.padding(18)
            .background(Theme.card)
            .clipShape(RoundedRectangle(cornerRadius: 18))
    }
}
